safe_transaction_service/history/management/commands/setup_service.py

```diff
@@ -362,6 +362,14 @@ def create_task(self) -> Tuple[PeriodicTask, bool]:
         ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 14062206, "1.3.0+L2"),
         ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 14062208, "1.3.0"),
     ],
+    EthereumNetwork.XDC: [
+        ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 54712487, "1.3.0+L2"),
+        ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 54712487, "1.3.0"),
+    ],
+    EthereumNetwork.TXDC_TESTNET: [
```

Comment:
XDC_MAINNET and XDC_TESTNET seems better.

Reply:
> @moisses89 Can we change TXDC_TESTNET to TXDC in [safe-eth-py ](https://github.com/safe-global/safe-eth-py/blob/master/gnosis/eth/ethereum_network.py#L46) ?
> 
> > @gzliudan I'll take a look to the chain name because I could see that [safe-eth-py ](https://github.com/safe-global/safe-eth-py/blob/master/gnosis/eth/ethereum_network.py#L46) currently use this value for the enum. Anyway the block number should be the block when the contract was deployed, @pkakelas should take a look to this.

I'll keep the name of safe-eth-py, that is using the chainlist names, we are going to update them shortly but keeping the chainlist.org names.

Reply:
https://chainlist.org/chain/51

Reply:
@gzliudan in this PR we are going to update the chain names to keep the same than ethereum list. https://github.com/safe-global/safe-eth-py/pull/452
XDC will be https://github.com/safe-global/safe-eth-py/pull/452/files#diff-bbae0e8b2a967b56299d26ef417fb5bebd62bef52760be4bd668e20bd19c0a5cR64